tests/frontend/test_kibana_app/cypress/integration/step-definitions/settings/modules/store-module-names.given.js

```diff
@@ -0,0 +1,5 @@
+Given('The user wants to activate the following modules', (dataTable) => {
+  storedModulesTable = dataTable
+});
+
+export let storedModulesTable;
```

Comment:
mmmm We should give this one more spin.

Reply:
I will investigate a better implementation for this